controllers/articles.controller.js

```diff
@@ -23,3 +24,14 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  return selectComments(article_id)
```

Comment:
need to check if an article exists before querying DB.  some articles have comments some don't. currently you are returning an error when an article exists